apps/web/src/routes/+layout.svelte

```diff
@@ -1,25 +1,75 @@
-<script>
+<script lang="ts">
 	import Sidebar from '$lib/components/sidebar/Sidebar.svelte';
 	import { fly } from 'svelte/transition';
 	import '../app.css';
 	import toast, { Toaster } from 'svelte-french-toast';
 	import Scans from '$lib/components/scans/Scans.svelte';
 	import { browser } from '$app/environment';
+	import { Focus } from 'lucide-svelte';
+	import { onMount } from 'svelte';
+	import { api } from '$lib/api/client';
+
+	let Device: any;
+
+	async function handleImageInput(event: InputEvent) {
+		console.log(event);
+		const formData = new FormData();
+		formData.append('img', event.target.files[0]);
+		formData.append('week_number', '4');
```

Comment:
Tohle by úplně nemělo být hardcodenutý, mělo by to být číslo týdne, ze kterého absence je

Reply:
Klidně bych zatím nechal hardocdnutý